c2corg_ui/static/js/apiservice.js

```diff
@@ -390,17 +390,18 @@ app.Api.prototype.updatePreferredLanguage = function(lang) {
  * @param {undefined | string} token
  * @param {string} lang
  * @param {boolean} isProfile
+ * @param {boolean} personalFeed
  * @return {!angular.$q.Promise<!angular.$http.Response>}
  */
-app.Api.prototype.readFeed = function(token, lang, isProfile) {
+app.Api.prototype.readFeed = function(token, lang, isProfile, personalFeed) {
   var url;
   var params = {'pl': lang};
   if (token) params['token'] = token;
 
   if (isProfile) {
     url = '/user-profile';
     params['u'] = this.auth_.userData.id;
-  } else if (this.auth_.isAuthenticated()) {
+  } else if (this.auth_.isAuthenticated() && personalFeed) {
```

Comment:
if you are logged and scroll through your personal-feed and then log out -> it will throw a big error on the screen. Checking for both allows to continue to scroll, but loading the normal /feed.